protocols/banklessvault.eth/events.json

```diff
@@ -1,7 +1,8 @@
-[
-  {
+{
+  "2021-05-14T15:00:00.000Z": {

```

Comment:
Is it possible to do this transform (date as object key) on the `hub-ui` side and keep `events.json` as a simple array of objects?

Reply:
ah yeah. I'll add it to the build step so hub-ui doesn't have to do anything, and the flow doesn't change for the perspective of whoever is adding these events